packages/next-app/src/app/metadata/[network]/pool/[poolId]/(components)/MetadataAttributesTable/PoolMetadataForm.tsx

```diff
@@ -104,28 +104,30 @@ export default function PoolMetadataForm({
           )}
         />
       </div>
-      <Input
-        label="Key"
-        placeholder={"Define an attribute key"}
-        disabled={mode === "edit"}
-        {...register("key")}
-        errorMessage={errors.key?.message}
-      />
+      <div className="flex flex-col gap-y-4">
```

Comment:
Curious about what motivated this change 👀

Reply:
The Input component had a hardcoded margin bottom 16px that was specific for pool metadata design. Now the Input has no margin and you have to set for each occasion you want to use. Does that make sense to you?